devtm.erip/lib/Money.php

```diff
@@ -11,7 +11,7 @@ public function setAmount($amount)
 		if(!is_numeric($amount))
 			throw new Exception("Price must be a number");
 		
-		$this->amount = $amount*100;
+		$this->amount = $amount;
```

Comment:
It is fine if you update `$allowed_currency = array("RUB", "BYR",	"USD", "EUR");` to `$allowed_currency = array("BYR");`